### ISSUE TITLE
Add subcription name to Article Object

### ISSUE THREAD

**Issue by jeena** [arturovm/pond#13]
In the Article Object there is subscription_id, it would make things much easier for feed reader developers if you would add the subscription name too there, or perhaps even the whole Subscription Object. It doesn't take much space but would allow to show every feed item without making another request to the server.

I had to implement a workaround for it in my reader for ownCloud News already so I will do that here too, but please consider this.

**Comment by arturovm** [comment]
Right, I see what you're saying. I had assumed that anybody who fetched articles would have had to fetch the user's subscription list first, and would therefore have a cached copy to reference the `id`s from. But I didn't account for users who fetch all articles without first fetching the subscription list.

I have to think of a solution that's as lightweight as possible.

**Comment by jeena** [comment]
But even if you fetch the subscription list first, in the meantime you perhaps add a new one which will not be in the cache, or if you change the name of the subscription, url, etc. Right now I am getting the whole subscription list every time before I get the unread articles just in case something changed.
